docker_install.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# This script installs Docker, copies the Dockerfile to the expressapi node app directory, builds and runs the Docker container.
+# Install Docker
+if ( which docker > /dev/null )
```

Comment:
Why just one assertion? Can this be broken up into better coverage of assertions to make it more idempotent?

Reply:
Yep, after our discussion on aws what you're saying here makes sense.